kafka/README.md

```diff
@@ -0,0 +1,43 @@
+# UPS and Apache Kafka 
+
+## Getting started with Docker
+### Linux
+Getting started with Linux is a fairly straightforward process:
+
+```
+$ docker run -d --name zookeeper --network kafka-net zookeeper:3.4
+$ docker run -d --name kafka --network kafka-net -p 9092:9092 --env ZOOKEEPER_IP=zookeeper ches/kafka
+```
+Connect to the Kafka broker by updating your `bootstrap.servers` with `localhost:9092` or the IP of the Kafka container.
+
+### Mac
+Establishing a connection between a container and a host service with Docker for Mac is slightly more convoluted. To get going, first run both the Zookeeper and Kafka containers normally:
+
+```
+$ docker run -d --name zookeeper --network kafka-net zookeeper:3.4
+$ docker run -d --name kafka --network kafka-net -p 9092:9092 --env ZOOKEEPER_IP=zookeeper ches/kafka
+```
+
+Get the container's IP address:
+
+```
+$ CONTAINER_IP=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kafka)
+$ echo $CONTAINER_IP 
+```
+Add this IP to the `loopback 0` interface: 
+```
+$ sudo ifconfig lo0 alias $CONTAINER_IP
```

Comment:
@dimitraz perhaps we add a little sentence here, like:

```
In addition to Linux, the following is required:
```

like an extra "warning" :warning: